userapi/storage/accounts/sqlite3/accounts_table.go

```diff
@@ -37,7 +37,9 @@ CREATE TABLE IF NOT EXISTS account_accounts (
     -- The password hash for this account. Can be NULL if this is a passwordless account.
     password_hash TEXT,
     -- Identifies which application service this account belongs to, if any.
-    appservice_id TEXT
+    appservice_id TEXT,
+    -- If the account is currently active
+    is_active BOOLEAN DEFAULT 1
```

Comment:
It's one of those personal preference things I'm sure, but I can't help but feel like this would make more sense flipped, e.g. `is_deactivated BOOLEAN DEFAULT 0`, in case we end up ever adding more fields like this.